### PR TITLE
feat:added badgeOnAvatar

### PR DESCRIPTION
**Pull request by GuptaEesh**
These badges on avatar contains :

**Differentiations:**
>Based on Sizes
   1. Small Avatar
   2. Medium Avatar
   3. Large Avatar
>Based on Modes
   1. Green(online)
   2. Red(offline)
   
badgeOnAvatar

### CODE REVIEW

**Review by abhij1607 — APPROVED** [review]
Seems Nice

File: components/badge-on-avatar.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    border: 1px solid var(--primary-400);
+    border-radius: 50%;
+    box-shadow: 0 0 5px var(--red-400);
+    height: 10px;
+    margin: 0;
+    position: absolute;
+    right: 0;
+    top: 5px;
+    width: 10px;
+}
```
Comment (Hamdrive) [inline components/badge-on-avatar.css:35]: Add new line at EOF

**Review by Hamdrive — COMMENTED** [review]
Good work done, ready to push

File: components/badge-on-avatar.css (hunk 1 above)
Comment (GuptaEesh) [reply components/badge-on-avatar.css]: Done✅

**Pull request merged**